thetis/coupled_timeintegrator.py

```diff
@@ -237,6 +238,8 @@ def _create_mom_integrator(self):
                   'uv_p1': self.fields.get('uv_p1_3d'),
                   'lax_friedrichs_velocity_scaling_factor': self.options.lax_friedrichs_velocity_scaling_factor,
                   'coriolis': self.fields.get('coriolis_3d'),
+                  'sipg_parameter': self.options.sipg_parameter,
```

Comment:
You are passing `solver.options.sipg_parameter` to the equations, so there's no need to add these parameters in `solver.fields`.